source/Glimpse.Site/Views/Shared/_Layout.cshtml

```diff
@@ -92,7 +92,7 @@
             <div class="row footer_last">
                 <div class="col-sm-3 footer_copy">&copy; @DateTime.Now.Year - Glimpse</div>
                 <div class="col-sm-6 footer_backing">
-                    Proudly backed by <a class="redgate" href="http://www.red-gate.com/"></a> <a href="http://www.github.com/glimpse/glimpse">and you!</a>
+                    Proudly backed by <a href="http://www.microsoft.com/"><img class="microsoft" src="~/Content/microsoft.png" alt="microsoft.com" width="125" height="29"></a> <a href="http://www.github.com/glimpse/glimpse">and you!</a>
```

Comment:
Not trying to be lazy here but does it HAVE to be a CSS fix? I'm getting the bottom 1/3 of the MS logo truncated so unless I introduce a few more divs/spans to manage individual elements, it may be too hard to get it with CSS alone. Unless my CSS-fu skills are not up to the challenge (hopefully they are). Feedback is welcome